src/features/contact-page/sections/ContactForm.js

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import Title from "@components/typography/Title";
+import { ThemeContext } from "@contexts/theme/ThemeContext";
+import { useTheme } from "@hooks/useTheme";
+
+import { contactFormStyles } from "../styles/ContactFormStyles";
+
+const ContactForm = () => {
+  const { t } = useTranslation();
+
+  const { themeState } = useTheme(ThemeContext);
+
+  return (
+    <section className={contactFormStyles[`${themeState.theme}`].section}>
+      <Title>{t("form-header")}</Title>
+      <form className={contactFormStyles[`${themeState.theme}`].form}>
+        <label className={contactFormStyles[`${themeState.theme}`].label}>
```

Comment:
Labelka z `p` i zawartością powtarza się w kilku miejscach. Możesz to wydzielić do osobnego komponentu (zmienia się input, który możesz przekazać jako children)